infra/cifuzz/fuzz_target.py

```diff
@@ -159,24 +159,35 @@ def is_reproducible(self, test_case, target_path):
         target_path: The path to the fuzz target to be tested
 
       Returns:
-        True if crash is reproducible.
+        (True, True) if crash is reproducible and we were able to run the
```

Comment:
Maybe a single return value of an enum is less error prone? Currently the first return value is a bit of a footgun.

Reply:
We should just raise an exception (e.g. `TargetRunError`) rather than trying to encode these cases, and keep the single return value. The caller can handle them accordingly and it's much clearer to follow the logic.

Reply:
Good point.
I think I'll also just do the check for the OSS-Fuzz target outside of this function and skip calling it if the target doesn't exist. I guess I don't like catching the exception (which could in theory be from the testcase not existing, though in practice I don't think this will ever happen) and assuming it was because of the OSS-Fuzz target being missing.

Reply:
I know this was a problem with the two return values as well. Not inherent to using exceptions.

Reply:
Done.